src/index.html

```diff
@@ -11,8 +11,56 @@
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
Ensure that the SCSS file is correctly compiled by the Parcel bundler. If there are any issues with SCSS compilation, check the configuration and paths.